src/client/components/inputs/index.tsx

```diff
@@ -45,7 +45,7 @@ const Wrapper = styled.div<{
 }>`
   position: relative;
   border-radius: 8px;
-  background-color: ${colorsV3.white};
+  background-color: ${colorsV3.gray100};
```

Comment:
I believe this will also affect the offer page and all current implementations of inputs fields. 
is that we want or is it better to override the background color on our file ?

Reply:
I asked Ellen about this and I think she said that they should be gray 🤔 But maybe it's safest to just change it in my component for now! Will fix that 😊

Reply:
Yes, maybe it's safest to just have them white for now, will change that!